webapp/commonComponents/expression/expressionVariables.js

```diff
@@ -8,37 +8,50 @@ import { sqlTypes } from '@common/surveyRdb/sqlTypes'
 import * as Expression from '@core/expressionParser/expression'
 
 // TODO: match all nodeDefTypes and throw an error if unknown:
-const toSqlType = nodeDef =>
-  NodeDef.isInteger(nodeDef) ? sqlTypes.integer : NodeDef.isDecimal(nodeDef) ? sqlTypes.decimal : sqlTypes.varchar
+const toSqlType = (nodeDef) => {
+  if (NodeDef.isInteger(nodeDef)) {
+    return sqlTypes.integer
+  }
+  if (NodeDef.isDecimal(nodeDef)) {
+    return sqlTypes.decimal
+  }
+  return sqlTypes.varchar
+}
+
+// Returns the name of the given nodeDef and appends "THIS" if it's equal to the current one
+const getVariableLabel = (nodeDef, nodeDefCurrent, i18n) =>
+  NodeDef.getName(nodeDef) +
+  (nodeDefCurrent && NodeDef.isEqual(nodeDef)(nodeDefCurrent) ? ` (${i18n.t('expressionEditor.this')})` : '')
 
-const getJsVariables = (nodeDef, lang) => [
+const getJsVariables = (nodeDef, nodeDefCurrent, i18n) => [
   {
     value: NodeDef.getName(nodeDef),
-    label: NodeDef.getLabel(nodeDef, lang),
+    label: getVariableLabel(nodeDef, nodeDefCurrent, i18n),
     type: toSqlType(nodeDef),
-    // TODO: add uuid here for symmetry?
+    uuid: NodeDef.getUuid(nodeDef),
   },
 ]
 
-const getSqlVariables = (nodeDef, lang) => {
+const getSqlVariables = (nodeDef, nodeDefCurrent, i18n) => {
   const colNames = NodeDefTable.getColNames(nodeDef)
 
   // TODO: Explain what getLabel does and why
```

Comment:
please, remove this old TODO. not sure who added it and why it's there

Reply:
ok